engine/gui/src/gui.cpp

```diff
@@ -1831,6 +1831,48 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
                         lua_setfield(L, -2, "gamepad_name");
                     }
 
+                    if (ia->m_HasGamepadPacket)
```

Comment:
This and the rest of the input handling code is line by line identical to the one in comp_script.cpp. Would it not make sense to move this shared code to some utility file and use it both in gui.cpp and comp_script.cpp? And at the same time ditch the InputAction struct in gui.h in favor of the one from gameobject,h (they are identical)

Reply:
Well, yes, that would be great. However, they are two separate libraries on the same "level", so the question is where to have the struct+functionality.